share/random_blobs.go

```diff
@@ -0,0 +1,64 @@
+package share
```

Comment:
How about to put test funcs into share_test package instead of main one?

Reply:
there are some cases where we are accessing private fields of the structs(not blob actually)

Reply:
Agreed to make to tackle it in separate PR